salt/modules/acme.py

```diff
@@ -91,6 +91,7 @@ def cert(name,
          email=None,
          webroot=None,
          test_cert=False,
+         certname=None,
```

Comment:
Can you move this new kwarg to the end of the list? We want to avoid an API change.